src/structures/Webhook.js

```diff
@@ -105,6 +105,19 @@ class Webhook {
 
     const { data, files } = await createMessage(this, options);
 
+    if (data.content instanceof Array) {
```

Comment:
Shouldn't we use `Array.isArray`? Even thought it has one occurence versus 26 occurences of `instanceof Array`.

Reply:
👍

Reply:
I don't see a reason why we would. `instanceof` is what we go with in this lib.